backend/repository/schemas/by_user.go

```
@@ -0,0 +1,26 @@
+	"polimane/backend/model"
+	awsdynamodb "polimane/backend/services/dynamodb"
+)
+
+func ByUser(ctx context.Context, user *model.User, attributes []string) ([]*model.Schema, error) {
+	var schemas []*model.Schema
+
+	query := awsdynamodb.Table().
+		Get("PK", user.ID).
+		Range("SK", dynamo.BeginsWith, model.SKSchema)
+
+	if attributes != nil && len(attributes) > 0 {
```

golangci-lint (1.64.8)

S1009: should omit nil check; len() for nil slices is defined as zero

(gosimple)